src/schema/objects/seo.js

```diff
@@ -66,14 +66,20 @@ export const seo = {
       ]
     },
     {
-      title: 'Advanced',
+      title: 'Geavanceerd',
```

Comment:
Laat em voor nu nog maar even op Engels staan in de library, als ie in het project aangepast moet worden kun je eventueel even het veld in het schema aanpassen.

Reply:
Het is nu een beetje gekke mix van Nederlands (`Markeren als cornerstone content`, `Probeer niet meer dan 156 karakters te gebruiken.`) en Engels. Vandaar de ~~change~~ wijziging.

Terug naar Engels?

Reply:
@peeke Jij mag kiezen wat het wordt, als het maar 1 taal is.

Reply:
Hmm, zit net te denken. We hebben sites (developers) waar Engels primair is. Engels is leesbaar voor nederlanders, maar andersom vaak niet. Ik zou dus inzetten op Engels en waar nodig vertalen in het project.

Als we zo'n vertaling doen kunnen we dat hier als optie gebruiken. En laten we hopen dat Sanity snel vertalingen toevoegt...